docs/guides/issue_triage.md

```diff
@@ -0,0 +1,135 @@
+| NXP          | Doru Gucea                              | [nxp](https://github.com/project-chip/connectedhomeip/issues?q=is%3Aopen+is%3Aissue+label%3Anxp)         |         |
+| Silabs/EFR32 | Jean Francois Penven, Junior Martinez   | [efr32](https://github.com/project-chip/connectedhomeip/issues?q=is%3Aopen+is%3Aissue+label%3Aefr32)     |         |
+
+## Code areas
+
+The following people can be contacted about issues in specific area of code are
+affected that are not platform-specific.
+
+| Code area                    | Contact                          | Label                                                                                            | Note(s)                                                                                                                              |
+| ---------------------------- | -------------------------------- | ------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
+| ARM cross-compile            | Andrei Litvin                    | N/A                                                                                              | Some support for cross-compiling on x64 linux for arm64 linux (like Raspberry PI)                                                    |
+| Build system (darwin, xcode) | Boris Zbarsky, Justin Wood       | N/A                                                                                              | darwin specific builds, xcode connector                                                                                              |
+| Build system (gn, ninja)     | Andrei Litvin                    | N/A                                                                                              | General build system questions and gn support, generally on Linux                                                                    |
+| MatterIDL                    | Andrei Litvin                    | N/A                                                                                              | General .matter files and code generation based on it                                                                                |
+| Minimal MDNS                 | Andrei Litvin                    | N/A                                                                                              | mdns specfic. Note that platformdns also exists and issues are often "DNSSD" rather than minmdns specific                            |
+| Python tests scripts         | Cecille Freeman                  | N/A                                                                                              | Writing tests cases in python                                                                                                        |
```

Comment:
selfish request - I'm not sure quite how to phrase this, but is there a way to clarify that all script bugs should not be routed to me - problems that are just problems with the scripts should go to the feature owners.

Reply:
Totally a legit concern. Tried to be more explicit in notes and description column, PTAL. I also think we should update it as needed if this still gets too many issues routed. I share a similar concern about `Linux` maintainance for myself (getting `does not work on this odd linux-based machine` ... I cannot maintain all linux environments for sure).